### PR TITLE
Gradle 1

### PR DESCRIPTION
**Pull request by JekaFRIZZ**

### CODE REVIEW

File: src/test/java/com/epam/utils/StringUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public void testIsPositiveNumberShouldReturnFalseWhenNegativeNumberApplied() {
+        boolean actual = StringUtils.isPositiveNumber("-100");
+        assertFalse(actual);
+    }
+
+    @Test
+    public void testIsPositiveNumberShouldReturnFalseWhenZeroApplied() {
+        boolean actual = StringUtils.isPositiveNumber("0");
+        assertFalse(actual);
+    }
```
Comment (andrey7067) [inline src/test/java/com/epam/utils/StringUtilsTest.java:24]: Add tests for null and a non-numeric value

**Review by andrey7067 — CHANGES_REQUESTED** [review]
